pkgs/top-level/all-packages.nix

```diff
@@ -12690,7 +12690,7 @@ with pkgs;
   gcc_latest = gcc11;
 
   # Use the same GCC version as the one from stdenv by default
-  gfortran = wrapCC (gccStdenv.cc.cc.override {
+  gfortran = wrapCC (gcc11.cc.override {
```

Comment:
Some context in https://github.com/NixOS/nixpkgs/pull/148539, need to find a much more proper fix.

Reply:
Instead of hardcoding `gcc11`, can we use `gcc` here? My intention was to make the unversioned gfortran attr resolve to the same version as the unversioned gcc attr.

Reply:
Let me do a rebuild to figure it out.